test/user.test.ts

```diff
@@ -0,0 +1,182 @@
+import { setupRecording, setupAPI } from "./setup";
+import { API_URI, TEST_AUTH_EMAIL, TEST_AUTH_PASSWORD } from "../src/config";
+import { USER } from "../src/apollo/queries";
+import { CountryCode } from "../src/apollo/types";
+import { saleorAuthToken } from "../src/core/constants";
+
+describe("user api", () => {
+  const context = setupRecording();
+  const { client, saleor } = setupAPI();
+  const testAddress = {
+    firstName: "Test name",
+    lastName: "Test lastname",
+    streetAddress1: "Test street address",
+    city: "Test city",
+    postalCode: "12-345",
+    country: "PL" as CountryCode,
+  };
+
+  beforeEach(async () => {
+    const { server } = context.polly;
+    server.any().on("beforePersist", (_, recording) => {
```

Comment:
WDYT bout extract this part from both files and don't duplicate?

Reply:
good catch